### ISSUE TITLE
Delete specific entries using upsertGraph

### ISSUE THREAD

**Issue by almahdi-chihaoui** [Vincit/objection.js#1063]
As I understand, currently, upsertGraph deletes/unrelates all the entries that are not presented. 
is there a way to target specific rows using their ids and some kind of option to delete only those rows without touching the other?
The reason for this is because in our situation, the frontend service cannot send all the data, it can only send the modified data.
Thanks.

**Comment by almahdi-chihaoui** [comment]
So, in the person example, if i don't want a movie with an id 5 to be deleted, the option should be: 
`{
    noDelete: ['movie.id == 5']
  }`
??

**Comment by koskimas** [comment]
No, that's not possible at the moment. This is actually related to #1061
